### PR TITLE
Add responsive mobile menu feature

### PR DESCRIPTION
**Pull request by MussieTeka**
This pull request adds a responsive mobile menu feature to the website. When the screen size is small enough to warrant the use of a mobile menu, the user can click the hamburger button to reveal the menu. They can then click any of the options in the menu to be taken to the corresponding section of the website.

Changes Made:

- Added a hamburger button icon to the top right of the header section
- Added a mobile menu that appears when the hamburger button is clicked
- Added a close (X) button to the top right of the mobile menu that closes the menu when clicked
- Added functionality to the mobile menu options to navigate to the correct section of the website when clicked

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -379,5 +392,14 @@ <h2>Contact me</h2>
...
       </form>
     </section>
+    <script>
+      var sidemenu = document.getElementById("sidemenu");
+      function openMenu() {
+        sidemenu.style.right = "0";
+      }
+      function closeMenu() {
+        sidemenu.style.right = "-375px";
+      }
```
Comment (ichala) [inline index.html:402]: - [ ] for [Javascript Best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) , kindly do not use a script tag to write your javascript code , instead  Keep all the JavaScript code in JS files , kindly fix it 🙏

File: index.html [hunk 2]
```diff
@@ -6,18 +6,27 @@
...
+        <nav>
+          <p class="logo">Mussie</p>
+          <ul id="sidemenu">
+            <li onclick="closeMenu()"><a href="#porfolio">Porfolio</a></li>
+            <li onclick="closeMenu()"><a href="#about-mobile">About</a></li>
+            <li onclick="closeMenu()"><a href="#contact">Contact</a></li>
+            <i id="close" class="fas fa-times" onclick="closeMenu()"></i>
+          </ul>
+          <i id="open" class="fas fa-bars" onclick="openMenu()"></i>
+        </nav>
```
Comment (ichala) [inline index.html:28]: - [ ] to match the figma design, kindly make sure the whole page is covered with the menu when it's open 🙏 

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Good design ✔️ 
- Menu Works ✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (MussieTeka) [reply index.html:28]: Great! Thank you for your feedback. I have made the necessary adjustments to ensure that the menu covers the entire page when opened, in order to match the Figma design. Thank you for bringing this to my attention!

**Review by franclobo — APPROVED** [review]
Hi @MussieTeka and @joeveloper, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️ 
- Descriptive PR ✔️ 
- All required changes completed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**